In this milestone I:

abelagza opened the pull request.

mahbubul14 reviewed (commented):
# Invalid Code Review Request

**Hi @abelagza ,** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project without `linter` file. Please add linter. I think it would be better if you add `.gitignore` file and add `node_modules/`in it. You should follow it for the project requirements.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

abelagza commented:
I added and fixed linters.

There seems to be 2 linters that I cannot get rid off, it is an indentantion issue. I added the 4 spaces the linter requires and won't fix it.
